spec/integration/spec_file_load_errors_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'support/aruba_support'
+require 'support/formatter_support'
+
+RSpec.describe 'Spec file load errors' do
+  include_context "aruba support"
+  include FormatterSupport
+
+  let(:failure_exit_code) { rand(97) + 2 } # 2..99
```

Comment:
Because I want the test to force the implementation to return the configured `failure_exit_code` and this seemed like the easiest way to do that.  Consider: if I just set `config.failure_exit_code` to a static value, an implementation that also returns that static value (but that does not reference `config.failure_exit_code`) could pass the test.  By using a random exit code between 2 and 99 it means that the only way the test can consistently pass is if the implementation really does use the configured value.